File: WorkIt-Server/WorkIt-Server/Models/ViewModels/ProfileDetailsViewModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace WorkIt_Server.Models.ViewModels
{
    public class ProfileDetailsViewModel
    {
        public int UserId { get; set; }
        public string Email { get; set; }
        public string FullName { get; set; }
        public string Phone { get; set; }
        public double RatingAsTasker { get; set; }
        public double RatingAsSupervisor { get; set; }
        public int NumberOfReviewsAsTasker { get; set; }
        public int NumberOfReviewsAsSupervisor { get; set; }
        public string PictureAsString { get; set; }
    }
}

File: WorkIt/app/src/main/java/prot3ct/workit/view_models/ProfileDetailsViewModel.kt
package prot3ct.workit.view_models

class ProfileDetailsViewModel (
    val userId: Int,
    val email: String,
    val fullName: String,
    val phone: String,
    val ratingAsTasker: Int,
    val ratingAsSupervisor: Int,
    val numberOfReviewsAsTasker: Int,
    val getNumberOfReviewsAsSupervisor: Int,
    val pictureAsString: String,
)